scripts/check-links.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
need to check those scripts on Windows or even disable?, including means that we limit an environment people can do their work

Reply:
ergh. Do you mean defining a `.huskyrc.js` which checks the current system/platform and according doesn't insert the check-links script? At that stage I may as well rewrite the whole checking script in `js` rather than `bash`.

Alternatively we could remove the hook completely but leave it as a CI test.

Reply:
yep, either of options is fine! just something really simple to not break the Windows folks experience

Reply:
well there's also a third option: python with the [regex](https://pypi.python.org/pypi/regex) module?

Reply:
ignoring in favour of CI-only (no pre-commit hook)